pkgs/games/animatch/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, allegro5
+, cmake
+, fetchFromGitLab
+, libGL
+, stdenv
+, xorg
+}:
+stdenv.mkDerivation rec {
+  pname = "animatch";
+  version = "1.0.3";
+  src = fetchFromGitLab {
+    owner = "HolyPangolin";
+    repo = "animatch";
+    fetchSubmodules = true;
+    rev = "v${version}";
+    hash = "sha256-zBV45WMAXtCpPPbDpr04K/a9UtZ4KLP9nUauBlbhrFo=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    allegro5
+    libGL
+    xorg.libX11
+  ];
+
+  cmakeFlags = [
+    "-DLIBSUPERDERPY_STATIC=ON"  # recommended by upstream for coexistence with other superderpy games
+  ];
+
+  meta = {
+    homepage = "https://gitlab.com/HolyPangolin/animatch/";
+    description = "a match-three game with cute animals";
+    longDescription = ''
+      The game was made for [Purism, SPC](https://puri.sm/)
```

Comment:
I think `longDescripition` is usually understandable without reading `description`, so it's better to mention that the game is match-3 somewhere in it.

Reply:
ah, thanks. i found a slightly better description from [this](https://github.com/dos1/animatch) mirror, which contains the key info i most wanted (that it's touch/mobile/phone-friendly), and updated it with that. i don't think there's really a suitable long description for this anywhere, so i'll just leave it blank.